package.json

```diff
@@ -204,6 +219,7 @@
       ],
       "no-bitwise": "off",
       "react/jsx-props-no-spreading": "warn",
+      "react/require-default-props": "off",
```

Comment:
Why was this disabled?

Reply:
removed it - it was being used for pathing but is unnecessary now.